homework_4/bets_api/internal/domain/models/betReduced.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+// BetReduced represents a domain model representation of a bet without CustomerId.
+type BetReduced struct {
+	Id                   string
+	Status               string
+	SelectionId          string
+	SelectionCoefficient float64
+	Payment              float64
+	Payout               float64
+}
```

Comment:
Why is this necessary?

Reply:
Because we do not want to return CustomerId. Maybe there is a better way to do so, but I could not think of any.

Reply:
We don't _need_ to or we _should not_ return CustomerId?

I believe it's the case of the former so there's no reason to overcomplicate things. Just returning the model with customer ID is fine 🤷

Reply:
I understood that we must not return CustomerId because of GDPR (something like that was mentioned in the class).

Reply:
Ah, alright, valid reason. It wasn't necessary to handle this in the homework though 😄 .

A cleaner way to do this would be to have a `BetHttpDto` model (or something similarly named) instead of `BetReduced` which only has the data we wish to return to the client, have controller receive full `Bet` model from the `BetService` and then map `Bet` -> `BetHttpDto` (possibly have a whole separate mapper class to do this).

That way you don't need to drag through the `BetReduced` model through multiple layers of the application and you have this "what do we want to return to the client" logic in one single place where it can be easily tested.